scan/src/components/ChainInfoHighlights.re

```diff
@@ -149,15 +151,19 @@ let make = () =>
           extraTopRight={<TimeAgos time={info.latestBlock.timestamp} />}
```

Comment:
Let's increase font size but you have to shorten like seconds to secs. @sorawit @taobun  Do you have any suggestions.

Reply:
As I've discussed with them, we just use `seconds` for now.